Makefile.kokkos

```diff
@@ -279,6 +285,18 @@ else
   KOKKOS_INTERNAL_OPENMPTARGET_FLAG := -fopenmp -foffload=nvptx-none
 endif
 
+# Set OpenACC flags.
+ifeq ($(KOKKOS_INTERNAL_COMPILER_PGI), 1)
+  KOKKOS_INTERNAL_OPENACC_FLAG := -O3 -acc
+else
+    ifeq ($(KOKKOS_INTERNAL_COMPILER_NVHPC), 1)
+      KOKKOS_INTERNAL_OPENACC_FLAG := -O3 -acc
```

Comment:
Why `-O3`?

Reply:
Yes, this is not the place for -O3. My fault!